purchase_delivery_split_date/readme/USAGE.rst

```diff
@@ -0,0 +1,11 @@
+[ This file must be present and contains the usage instructions
```

Comment:
Can you remove or change it to an actual description on how to use the module :smile: .